ui/src/stores/HomeStore.js

```diff
@@ -406,14 +406,17 @@ class HomeStore {
 
   @action
   async filterByTxHashInReturnValues(transactionHash) {
-    const { blockNumber } = await this.homeWeb3.eth.getTransaction(transactionHash)
-    const events = await this.getEvents(blockNumber, blockNumber)
+    const events = await this.getEvents(1, 'latest')
```

Comment:
Does it means that the startBlock is `1`? Do you think it makes sense to choose more relevant block? E.g. a block that was mined a day ago from the current moment or something like this?

Reply:
But what if a user want to access the events for the transaction happened more than 1 day ago?

Reply:
Is this for scenario when a user accesses a tx in the event tab? Will this tab display not mined transactions? If it is not from this tab, how the will the user access this transaction?

Reply:
This function is used for searching home side events which include transactionHash on the foreign side in their data (such as `AffirmationCompleted`, `SignedForAffirmation`)